packages/roosterjs-content-model/lib/modelApi/creators/createListItem.ts

```diff
@@ -8,14 +8,19 @@ import { createSelectionMarker } from './createSelectionMarker';
  */
 export function createListItem(
     levels: ContentModelListItemLevelFormat[],
-    format?: ContentModelSegmentFormat
+    format?: ContentModelSegmentFormat,
+    textAlign?: string
```

Comment:
As I commented in another place, this should be done by format handler, but not through a parameter of creator.

Reply:
One of the reason is, here you allow to pass a string, then later you convert the type to "start|end|center". What if people pass in some other string? We can't detect it and there is no build time error. So that will most likely cause bug. To solve it, people need to convert their string to the type we need. But that is already done in format handler, so it is a dup.